lms/djangoapps/discussion_api/serializers.py

```diff
@@ -182,7 +186,7 @@ class ThreadSerializer(_ContentSerializer):
     endorsed_comment_list_url = serializers.SerializerMethodField("get_endorsed_comment_list_url")
     non_endorsed_comment_list_url = serializers.SerializerMethodField("get_non_endorsed_comment_list_url")
 
-    non_updatable_fields = ("course_id",)
+    non_updatable_fields = NON_UPDATABLE_THREAD_FIELDS
```

Comment:
`_ContentSerializer.non_updatable_fields` is a `tuple`, but here it's getting assigned it to a `set`. Please either convert to a tuple here, or change `_ContentSerializer.non_updatable_fields` to be initialized to `set()`